python/pyspark/ml/feature.py

```diff
@@ -437,33 +498,20 @@ class CountVectorizer(JavaEstimator, HasInputCol, HasOutputCol, JavaMLReadable,
     >>> loadedModel = CountVectorizerModel.load(modelPath)
     >>> loadedModel.vocabulary == model.vocabulary
     True
+    >>> fromVocabModel = CountVectorizerModel.from_vocabulary(model.vocabulary,
```

Comment:
This might be better with an explicit manual array rather than model.vocabulary to show folks how to expect to use it? What are your thoughts?

Reply:
Yeah, totally agree let me change it